cmd/expiration-mailer/main_test.go

```diff
@@ -70,7 +70,7 @@ func newFakeClock(t *testing.T) clock.FakeClock {
 	return fc
 }
 
-const testTmpl = `hi, cert for DNS names {{.DNSNames}} is going to expire in {{.DaysToExpiration}} days ({{.ExpirationDate}})`
+const testTmpl = `hi, certs {{if .DNSSerials}}{{.DNSSerials }} {{end}}for DNS names {{.DNSNames}} is going to expire in {{.DaysToExpiration}} days ({{.ExpirationDate}})`
```

Comment:
Is there a reason to put the `{{.DNSSerials}}` in conditionally? Shouldn't there always be values?